src/sidebar/util/time.js

```diff
@@ -110,37 +104,39 @@ function dayAndMonthAndYear(date, now, Intl) {
 
 const BREAKPOINTS = [
   {
-    test: lessThanThirtySecondsAgo,
-    format: function() {
-      return 'Just now';
-    },
-    nextUpdate: 1,
+    // Less than 30 seconds
```

Comment:
One thing we lose by making the tests anonymous inline is that they were named nicely before. The test functions are fairly self-documenting, I hope?, but I left some comments for each, too.